configs and readmes for vLLM openAI-compatible ultravox 0.4 and pixtral 12b (#361)
basetenlabs/truss-examples: trussless/README.md

> ## Deploying on Baseten Without Truss

Reviewer: ```suggestion
## Deploying customer server to Baseten
```

If you've deployed on [Baseten](https://baseten.co) before, you're likely familiar with using [Truss](https://docs.baseten.co/quickstart) to do so.

> This process typically involves creating a `model.py` file, which contains the code for your model, and packaging it for deployment via an API endpoint managed by Baseten.

Reviewer: ```suggestion
This process typically involves creating a `model.py` file, which contains the code for your model, and packaging it for deployment via an API endpoint managed by Baseten. Internally, `truss` creates a server and deploys it to provide this API endpoint.
```

> However, there are situations where you might want to deploy a model that's already wrapped in an API. A common example is the `vLLM` OpenAI-Compatible Server, which provides its own HTTP endpoint. Another scenario is the "bring your own image" approach, where you have a Docker image that can handle HTTP requests directly.

Reviewer: ```suggestion
However, there are situations where you might want to deploy a model that's already wrapped in an API using a custom server. A common example is the `vLLM` OpenAI-Compatible Server, which provides its own HTTP endpoints. Another scenario is the "bring your own image" approach, where you have a Docker image that can handle HTTP requests directly.
```
